grakn-test/test-integration/src/test/java/ai/grakn/test/graknmodule/GraknModuleTest.java

```diff
@@ -0,0 +1,74 @@
+package ai.grakn.test.graknmodule;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.FilenameUtils;
+import ai.grakn.engine.module.GraknModuleManager;
+import ai.grakn.graknmodule.GraknModule;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.List;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import static org.junit.Assert.assertEquals;
+
+public class GraknModuleTest {
+    private final String DUMMY_MODULE_NAME = "dummy-module";
+    private final String DUMMY_MODULE_JAR_URL = "https://github.com/lolski/grakn-dummy-module/raw/master/dummy-module-1.0-SNAPSHOT.jar";
```

Comment:
I thought we were going to add this jar to the repo? If we don't do that we should maybe add it to a commonly accessible repo  rather than keep it on your personal repo.

Reply:
I remember that someone mentioned that most of the time we refer to such fixture from external resources, so I figured that we should stay consistent with the approach.

I agree with moving it under `graknlabs`, any suggestions of which repo to use?

Reply:
Alright. I personally would not mind having this jar in the main repo.

We don't have any repo right now which could house that specific jar. We will need to create a new one and for the sake of a single jar I am kind of meh about it.

Reply:
We do need to house the `dummy-module`'s source code somewhere as well so we can update the JAR. The module SDK will grow in functionality and the test / JAR would need to be updated as well.